pennylane/ftqc/lightning_backend.toml

```diff
@@ -0,0 +1,68 @@
+schema = 3
+
+# The set of all gate types supported at the runtime execution interface of the
+# device, i.e., what is supported by the `execute` method of the Device API.
+# The gate definition has the following format:
+#
+#   GATE = { properties = [ PROPS ], conditions = [ CONDS ] }
+#
+# where PROPS and CONS are zero or more comma separated quoted strings.
+#
+# PROPS: zero or more comma-separated quoted strings:
+#        - "controllable": if a controlled version of this gate is supported.
+#        - "invertible": if the adjoint of this operation is supported.
+#        - "differentiable": if device gradient is supported for this gate.
+# CONDS: zero or more comma-separated quoted strings:
+#        - "analytic" or "finiteshots": if this operation is only supported in
+#          either analytic execution or with shots, respectively.
+#        - "terms-commute": if this composite operator is only supported
+#          given that its terms commute. Only relevant for Prod, SProd, Sum,
+#          LinearCombination, and Hamiltonian.
+#
+[operators.gates]
+
+CNOT                   = { }
+GlobalPhase            = { }
+Hadamard               = { }
+Identity               = { }
+PauliX                 = { }
+PauliY                 = { }
+PauliZ                 = { }
+RotXZX                 = { }
+RZ                     = { }
+S                      = { }
+
+
+# Observables supported by the device
+[operators.observables]
+
+
+[measurement_processes]
+
+SampleMP               = { conditions = [ "finiteshots" ] }
+
+
+# Additional support that the device may provide. All accepted fields and their
+# default values are listed below. Any fields missing from the TOML file will be
+# set to their default values.
+[compilation]
+
+# Whether the device is compatible with qjit.
+qjit_compatible = true
+
+# Whether the device requires run time generation of the quantum circuit.
+runtime_code_generation = false
+
+# The methods of handling mid-circuit measurements that the device supports, e.g.,
+# "one-shot", "device", "tree-traversal", etc. An empty list indicates that the device
+# does not support mid-circuit measurements.
+supported_mcm_methods = [ "one-shot" ]
+
+# Whether the device supports dynamic qubit allocation/deallocation.
+dynamic_qubit_management = false
```

Comment:
I guess this needs to be true for the eventual system, but fine for false in our case for LQ, given the support that exists.